.github/workflows/e2e.yaml

```
@@ -4918,70 +4921,39 @@ jobs:
-      E2E_JOB: "1"
-      E2E_TARGET_ID: "jetson-nvmap-gpu"
-      E2E_ARTIFACT_DIR: ${{ github.workspace }}/e2e-artifacts/live/jetson-nvmap-gpu
-      NEMOCLAW_CLI_BIN: ${{ github.workspace }}/bin/nemoclaw.js
-      NEMOCLAW_RUN_LIVE_E2E: "1"
-      NEMOCLAW_NON_INTERACTIVE: "1"
-      NEMOCLAW_ACCEPT_THIRD_PARTY_SOFTWARE: "1"
-      NEMOCLAW_SANDBOX_NAME: "e2e-jetson-nvmap"
-      NEMOCLAW_RECREATE_SANDBOX: "1"
-      NEMOCLAW_PROVIDER: "ollama"
-      OPENSHELL_GATEWAY: "nemoclaw"
+      E2E_ARTIFACT_DIR: ${{ runner.temp }}/e2e-artifacts/live/jetson-nvmap-gpu
```

actionlint (1.7.12)

[error] context "runner" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "secrets", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)